db/schema.rb

```diff
@@ -19,7 +19,7 @@
     t.string "record_type", null: false
     t.bigint "record_id", null: false
     t.bigint "blob_id", null: false
-    t.datetime "created_at", null: false
+    t.datetime "created_at", precision: nil, null: false
```

Comment:
A load of these kind of things have changed in the schema, missing precisions etc

Reply:
Oh hang on, I saw that had changed (no idea why) but I thought I'd excluded it from the commit. Will fix.